test/integration/k8s_test.go

```diff
@@ -184,10 +184,34 @@ func TestPodScaling(t *testing.T) {
 				}
 				t.Log("all pods have been allocated IPs")
 			}) {
-				errors.New("Pods don't have IP's")
-				return
+				t.Error("some pods don't have IPs")
 			}
 
+			t.Run("nodenetworkconfig has converged", func(t *testing.T) {
+				clusterCheckCtx, cancel := context.WithTimeout(ctx, 20*time.Minute)
```

Comment:
Would it be worthwhile to bind this to the test suite's timeout as well? That way it's the lesser of that timeout and 20 minutes?